vips/vip-193/vip-193-testnet.ts

```diff
@@ -0,0 +1,222 @@
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: false,
+    isMock: true,
+  },
+  {
+    name: "vXVS",
+    address: "0x6d6F697e34145Bb95c54E77482d97cc261Dc237E",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: true,
+    isMock: false,
+  },
+  {
+    name: "vDOGE",
+    address: "0xF912d3001CAf6DC4ADD366A62Cc9115B4303c9A9",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: true,
+    isMock: true,
+  },
+  {
+    name: "vMATIC",
+    address: "0x3619bdDc61189F33365CC572DF3a68FB3b316516",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: true,
+    isMock: true,
+  },
+  {
+    name: "vCAKE",
+    address: "0xeDaC03D29ff74b5fDc0CC936F6288312e1459BC6",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: true,
+    isMock: true,
+  },
+  {
+    name: "vADA",
+    address: "0x37C28DE42bA3d22217995D146FC684B2326Ede64",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: false,
+    isMock: true,
+  },
+  {
+    name: "vAAVE",
+    address: "0x714db6c38A17883964B68a07d56cE331501d9eb6",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: false,
+    isMock: true,
+  },
+  {
+    name: "vBUSD",
+    address: "0x08e0A5575De71037aE36AbfAfb516595fE68e5e4",
+    reduceReservesBlockDelta: 100,
+    acceptAdmin: false,
+    isMock: true,
+  },
+];
+
+function filterAssets(assets: AssetConfig[]) {
+  return assets.filter(asset => asset.acceptAdmin);
+}
+
+const MARKETS_WITH_ACCEPT_ADMIN = filterAssets(CORE_MARKETS);
+
+export const vip193Testnet = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-193 VToken Upgrade of AIA Part - 2",
+    description: `upgrade the implementation of the Vtoken core supportimg Automatic income allocation feature.`,
+    forDescription: "I agree that Venus Protocol should proceed with VToken Upgrade of AIA",
+    againstDescription: "I do not think that Venus Protocol should proceed with VToken Upgrade of AIA",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds with VToken Upgrade of AIA or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "setReduceReservesBlockDelta(uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "_setReserveFactor(uint256)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "_setInterestRateModel(address)", NORMAL_TIMELOCK],
+      },
+      {
+        target: ACCESS_CONTROL_MANAGER,
+        signature: "giveCallPermission(address,string,address)",
+        params: [ethers.constants.AddressZero, "_reduceReserves(uint256)", NORMAL_TIMELOCK],
+      },
```

Comment:
I would copy these three permissions to VIP-192, to make it self-contained. I'm not sure if we have to keep them to make the simulation works